src/uct/ib/dc/dc_mlx5.h

```diff
@@ -74,7 +74,10 @@ typedef enum {
     UCT_DC_MLX5_IFACE_FLAG_KEEPALIVE_FULL_HANDSHAKE = UCS_BIT(1),
 
     /** uidx is set to dci idx */
-    UCT_DC_MLX5_IFACE_FLAG_UIDX                     = UCS_BIT(2)
+    UCT_DC_MLX5_IFACE_FLAG_UIDX                     = UCS_BIT(2),
+
+    /** Flow control endpoint failed */
```

Comment:
/** Flow control endpoint is using a DCI in error state */

Reply:
done